<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidechainWTPrimeDialog</class>
 <widget class="QDialog" name="SidechainWTPrimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sidechain Withdrawal (WT^) Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableViewWTPrimeVote"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonUpvote">
       <property name="text">
        <string>Upvote</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAbstain">
       <property name="text">
        <string>Abstain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDownvote">
       <property name="text">
        <string>Downvote</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Set a default vote for all WT^(s). Disabled if custom votes are set.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClearVotes">
     <property name="text">
      <string>Reset WT^ votes with the RPC command 'clearwtprimevotes'</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxDefaultVote">
     <item>
      <property name="text">
       <string>Upvote</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Abstain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Downvote</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
